.gitignore

```diff
@@ -55,12 +55,12 @@ build-iPhoneSimulator/
 # Used by RuboCop. Remote config files pulled in from inherit_from directive.
 # .rubocop-https?--*
 
-# SOA-specific
-config/secrets.yml
```

Comment:
IMPORTANT
`secretes.yml` should not upload to version control
and keep this line in gitignore.

I suggest you provide a `secrets_example.yml `with necessary real key and fake key. refers to professor's file. [proffersor's file](https://github.com/ISS-SOA/soa2021-codepraise/blob/main/config/secrets_example.yml)

Reply:
We did not put out real credentials in the `secrets.yml`, so it seem that we just need to rename it to `secrets_example.yml`, thank you for reminding us!